server/routes/editTag.ts

```diff
@@ -4,37 +4,43 @@ import lookupCommunityIsVisibleToUser from '../util/lookupCommunityIsVisibleToUs
 import { factory, formatFilename } from '../util/logging';
 const log = factory.getLogger(formatFilename(__filename));
 
+export const Errors = {
+  NotLoggedIn: 'Not logged in.',
+  NoTagId: 'Must supply tag ID.',
+  NotAdmin: 'Must be an admin to feature tags.',
+  TagNotFound: 'Tag not found.'
+};
+
 const editTag = async (models, req: Request, res: Response, next: NextFunction) => {
   const [chain, community] = await lookupCommunityIsVisibleToUser(models, req.body, req.user, next);
   if (!req.user) {
-    return next(new Error('Not logged in'));
+    return next(new Error(Errors.NotLoggedIn));
   }
   if (!req.body.id) {
-    return next(new Error('Must supply tag ID'));
-  }
-  if (req.body.featured_order) {
-    const adminAddress = await models.Address.findOne({
-      where: {
-        address: req.body.address,
-        user_id: req.user.id,
-      },
-    });
-    const roleWhere = {
-      address_id: adminAddress.id,
-      permission: 'admin',
-    };
-    if (community) roleWhere['offchain_community_id'] = community.id;
-    else if (chain) roleWhere['chain_id'] = chain.id;
-    const requesterIsAdminOrMod = await models.Role.findAll({
-      where: roleWhere,
-    });
-    if (!requesterIsAdminOrMod) return next(new Error('Must be an admin to feature tags.'));
+    return next(new Error(Errors.NoTagId));
   }
 
+  const adminAddress = await models.Address.findOne({
+    where: {
+      address: req.body.address,
+      user_id: req.user.id,
+    },
+  });
+  const roleWhere = {
+    address_id: adminAddress.id,
+    permission: 'admin',
+  };
+  if (community) roleWhere['offchain_community_id'] = community.id;
```

Comment:
can we wrap these in brackets? seems we're doing that with most other conditionals.

Reply:
Sure, do you want lines 43-45 bracketed out too? Should we add to the linter, if we wanna get rid of one-line conditionals entirely?

Reply:
I think one liners are fine if they exist alone. If we have multiple conditional cases then we should use brackets.